src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -55,6 +61,124 @@
         Locale.ENGLISH,
     };
 
+    private static final Map<String, Set<String>> TOKENS_IN_CONFIG_TO_IGNORE = new HashMap<>();
+
+    static {
+        TOKENS_IN_CONFIG_TO_IGNORE.put("NoWhitespaceBefore",
```

Comment:
unfortunately we need to make comment on each IGNORE, to explain reason, it might be clear now , but it will not be clear in time.
Please do.